deploy/olm-catalog/apicurio-registry/0.0.5-dev/apicurio-registry.v0.0.5-dev.clusterserviceversion.yaml

```diff
@@ -133,7 +133,7 @@ spec:
             x-descriptors:
               - urn:alm:descriptor:com.tectonic.ui:fieldGroup:ui
               - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
-          - description: Override the image name according to selected Persistence Type for example, MEM="docker.io/apicurio/apicurio-registry-mem:latest" KAFKA="docker.io/apicurio/apicurio-registry-kafka:latest" STREAMS="docker.io/apicurio/apicurio-registry-streams:latest" JPA="docker.io/apicurio/apicurio-registry-jpa:latest" INFINISPAN="docker.io/apicurio/apicurio-registry-infinispan:latest"
+          - description: Override the image name according to selected Persistence Type for example, MEM="quay.io/apicurio/apicurio-registry-mem:latest" KAFKA="quay.io/apicurio/apicurio-registry-kafka:latest" STREAMS="quay.io/apicurio/apicurio-registry-streams:latest" JPA="quay.io/apicurio/apicurio-registry-jpa:latest" INFINISPAN="quay.io/apicurio/apicurio-registry-infinispan:latest"
```

Comment:
This part originates from the CSV template, but I can fix it quickly after merge.